src/FsAutoComplete.Core/InlayHints.fs

```diff
@@ -80,16 +80,9 @@ type private FSharp.Compiler.CodeAnalysis.FSharpParseFileResults with
             | SynExpr.Typed(_expr, _typeExpr, range) when Position.posEq range.Start pos -> Some range
             | _ -> defaultTraverse expr
 
-          override _.VisitSimplePats(_path, pats) =
-            match pats with
-            | [] -> None
-            | _ ->
-              let exprFunc pat =
-                match pat with
-                | SynSimplePat.Typed(_pat, _targetExpr, range) when Position.posEq range.Start pos -> Some range
-                | _ -> None
-
-              pats |> List.tryPick exprFunc
+          override visitor.VisitSimplePats(_path, pat) =
```

Comment:
I asked about this API change in slack - I'm not sure that VisitSimplePats should have changed here. This is echoed a few places in the rest of the changes in this commit.